src/common/components/layouts/pages/results/summary/Summary.js

```diff
@@ -88,35 +80,25 @@ function TimeoutSummary() {
     );
 }
 
-function buildChartData({ passedChecks, failedChecks }) {
-    return [
-        ['Check', 'Number'],
-        ['Passed', passedChecks],
-        ['Error', failedChecks],
-    ];
-}
-
-function getChartOptions(theme) {
-    return {
-        pieHole: 0.7,
-        title: '',
-        slices: [{ color: theme.palette.success.main }, { color: theme.palette.error.main }],
-        legend: 'none',
-        pieSliceText: 'none',
-        height: '300px',
-        width: '300px',
-        chartArea: {
-            height: '80%',
-        },
-    };
-}
-
-function calculateCompliance({ passedChecks, failedChecks }) {
-    const total = passedChecks + failedChecks;
-    if (total === 0) {
-        return 100;
-    }
-    return Math.floor((passedChecks * 100) / total);
+function getListOfErrors(ruleSummaries) {
+    const listOfErrors = {};
+    const otherIndexes = [];
+    const categoryTags = errorTags[CATEGORY].map(({ name }) => name);
+    categoryTags.forEach(key => {
+        listOfErrors[key] = ruleSummaries
+            ?.filter((rule, index) => {
+                if (_.isNil(rule)) return false;
+                // Avoid duplicated rules in different categories
+                const isRuleShow = rule.tags?.includes(key) && !otherIndexes.includes(index);
+                if (isRuleShow) {
+                    otherIndexes.push(index);
+                }
+                return isRuleShow;
+            })
+            .reduce((num, item) => num + item.failedChecks, 0);
+    });
+    const sortedList = _.fromPairs(_.sortBy(_.toPairs(listOfErrors), 1).reverse());
```

Comment:
I'd suggest to create a list of objects { category, amout } or list of pairs [ category, amount ] to get rid redundant actions (we create object, convert it to array to order, convert back to object and convert to array when render).